{{cookiecutter.project_slug}}/setup.py

```diff
@@ -16,7 +16,17 @@
     # TODO: put package requirements here
 ]
 
+setup_requirements = [
+{%- if cookiecutter.use_pytest == 'y' %}
+    pytest-runner
```

Comment:
By the way, isn't that ``run_inside_dir(...) == 0`` line supposed to be prefixed by an ``assert``?  Simple comparisons without assertion won't make tests fail even if they fail….